app/javascripts/gencat/modules/gp_gencat_departments_controller.js

```diff
@@ -19,24 +19,55 @@ window.GobiertoPeople.GencatDepartmentsController = (function() {
   }
 
   GencatDepartmentsController.prototype.show = function(options) {
-    _loadRowchart('#department_people_events_rowchart', options.department_people_events_rowchart_api_path)
-    _loadRowchart('#department_interest_groups_events_rowchart', options.department_interest_groups_rowchart_api_path)
-    _loadPunchcard(
-      '#department_people_events_punchcard',
-      options.department_people_events_punchcard_api_path,
-      I18n.t('gobierto_people.departments.show.punchcard_title')
-    )
-
-    _reloadRowchart('#department_people_events_rowchart', options.department_people_events_rowchart_api_path, 10000)
-    _reloadRowchart('#department_interest_groups_events_rowchart', options.department_interest_groups_rowchart_api_path, 10000)
-
-    // REVIEW: Waiting for render
-    setTimeout(function () {
-      setTooltipColor()
-    }, 1000);
+    const peopleBoxes = document.querySelector(".js-people-rectangles")
+    setPeopleBoxes(peopleBoxes, options.department_people_events_rowchart_api_path)
   };
 
   return GencatDepartmentsController;
 })();
 
+function setPeopleBoxes(element, url) {
+    // notice that "key, value & url" are properties of the API response object
+    const template = `
+      <div class="rectangle">
+        <div class="rectangle--inner">
+          <div class="rectangle--content">
+            <div class="rectangle--content-inner">
+              <a href="{{ url }}">
+                <h1 class="rectangle--title"><strong>{{ name }}</strong></h1>
+              </a>
+              <div class="rectangle--subtitle">{{ position }}</div>
+            </div>
+          </div>
+        </div>
+      </div>
+    `;
+
+    const emptyTemplate = `<div class="col-md-12">${I18n.t("gobierto_people.shared.noresults")}</div>`;
+
+    // get initial data
+    const endpoint = appendUrlParam(url, "limit", 1000)
```

Comment:
This is pretty weak and might cause some inconsistencies in the future if new titles are added. I saw that now JS have new classes to remove accents and special chars from strings: https://stackoverflow.com/questions/990904/remove-accents-diacritics-in-a-string-in-javascript

Reply:
Ok. Regarding `getSortingKey` function. You mean normalize the value instead of a specific regex? In such case, the regex would be quite similar, *conseller[a]* will keep the same, *secretari[a] general*, will change only one letter

I'm using the normalization at the `input` event from the search box, to take a look.

Reply:
Function improved